codec-http2/src/test/java/io/netty/handler/codec/http2/Http2CodecTest.java

```diff
@@ -110,10 +111,10 @@ public void createOutboundStream() {
         Http2Headers headers = new DefaultHttp2Headers();
         childChannel.write(new DefaultHttp2HeadersFrame(headers));
         ByteBuf data = Unpooled.buffer(100).writeZero(100);
-        childChannel.writeAndFlush(new DefaultHttp2DataFrame(data));
+        childChannel.writeAndFlush(ReferenceCountUtil.releaseLater(new DefaultHttp2DataFrame(data)));
```

Comment:
The write operation will take reference count ownership, and thus release the object. Shouldn't we retain here, and release after we verify the data later, otherwise we may double release.